content/300-guides/100-performance-and-optimization/090-metrics.mdx

```diff
@@ -0,0 +1,335 @@
+---
+title: 'Metrics'
+metaTitle: 'Metrics'
+metaDescription: 'Diagnose application performance with insights into Prisma Client database activity'
+tocDepth: 3
+---
+
+<TopBlock>
+
+Prisma metrics give you a detailed insight into how Prisma Client interacts with your database. You can use this insight to help diagnose performance issues with your application. If you send the data to an external metrics system, then you can view the data over time. For example, you might visualize how your application's number of idle and active connections change.
+
+</TopBlock>
+
+## About metrics
+
+Metrics provide information on the following aspects of Prisma Client:
+
+- The number of Prisma operations
+
+  A Prisma operation is a statement like `let user = prisma.user(include: {posts: true}})`, which Prisma Client translates into one or more SQL queries.
+
+- Query response times
+- Active transactions
+- Connection pool information
+
+You can export metrics in JSON or Prometheus formats and view them in a console log, or integrate them into an external metrics system, such as StatsD or Prometheus. If you integrate them into an external metrics system, then you can view the metrics data over time, for example to help diagnose changes in your application's performance.
+
+We provide the following metrics:
+
+- Counters (always increase):
+
+  - `query_total_queries`: The total number of database queries executed.
+  - `query_total_operations`: The total number of operations executed.
+
+    This can be less than `query_total_queries`, because some operations create multiple queries.
+
+- Gauges (can increase or decrease):
+  - `pool_active_connections`: The number of active connections in use.
+  - `pool_idle_connections`: The number of connections that are not being used.
+  - `pool_wait_count`: The number of workers waiting for a connection.
+  - `query_active_transactions`: The number of active transactions.
+- Histograms (distributions of values categorized into "buckets"):
+  - `pool_wait_duration_ms`: The wait time for a worker to get a connection.
+  - `query_total_elapsed_time_ms`: The execution time for all database queries executed.
+  - `query_operation_total_elapsed_time_ms`: The execution time for all operations.
+
+You can group your metrics data with [global labels](#global-labels).
+
+## Get metrics in JSON format
+
+When you get metrics in JSON format, you can use them as returned, or [send them to StatSD](#use-prisma-metrics-with-statsd).
```

Comment:
One thing I've forgotten to mention is that because metrics is a preview feature. They will need to add `"metrics"` to the list of `previewFeatures` in their schema before it will work